src/nft/components/collection/ActivityCells.tsx

```diff
@@ -57,6 +58,15 @@ const AddressLink = styled(ExternalLink)`
   }
 `
 
+const isPurchasableOrder = (orderStatus?: OrderStatus, marketplace?: string): boolean => {
+  if (!marketplace || !orderStatus) return false
+  const purchasableMarkets = Object.keys(NftMarketplace).map((market) => market.toLowerCase())
```

Comment:
this seems to include all markets, is the plan to note the non-puchaseable Nfts in a later PR?

Reply:
actually this is not quite the case I talked to mz about this and he said that the enum for `NftMarketplace` is only used for purchasable markets thats why this works, but our enum for markets has all of the markets in it

Reply:
thats why it is actually done like this - previously it was an array until mz informed me of this

Reply:
is there any chance this would ever change on the backend? it sounds a bit brittle to me because the object is `NftMarketplace` but there is an assumption you'll only get purchasable ones. i think either naming it `PurchasableNftMarketplace` or adding a purchasable field per marketplace could help - or is there any other way to get this info?

Reply:
Going to move this to another pr, but I talked to BE and this is what they define as purchasable on their side. So it might change but that means something is now purchasable or not so I think the fact that it could change is good